src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostTaskSearchControllerTest.java

```diff
@@ -513,7 +513,8 @@ void should_return_400_bad_request_when_work_types_did_not_exist() throws Except
                                + "decision_making_work, applications, priority, access_requests, "
                                + "error_management, review_case, evidence, follow_up, pre_hearing, post_hearing, "
                                + "intermediate_track_hearing_work, multi_track_hearing_work, "
-                               + "intermediate_track_decision_making_work, multi_track_decision_making_work]")
+                               + "intermediate_track_decision_making_work, multi_track_decision_making_work, "
+                               + "query_work]")
```

Comment:
It would be good to add a test which uses the new Work Type as a filter attribute to the searches.  I expect there is already a test somewhere that does this with other Work Types so you may be able to extend it and use this work type.
This particular test is just checking that the 400 error includes query_work as one of the acceptable Work Types - so not using it as a filter.

Reply:
The same test above can verify that the new Work Type id is included in the search response.